src/plugins/dashboard/public/application/lib/migrate_app_state.ts

```diff
@@ -29,8 +31,9 @@ import {
   SavedDashboardPanel640To720,
   SavedDashboardPanel620,
   SavedDashboardPanel,
-} from '../types';
-import { migratePanelsTo730 } from '../../migrations/migrate_to_730_panels';
+} from '../../types';
+// should be moved in src/plugins/dashboard/common right after https://github.com/elastic/kibana/pull/61895 is merged
```

Comment:
Could you please clarify? this is a link to the same pr 🤔 
circular dependency :D

Reply:
Ah, yeah, awkward.
An infinite loop 😆 
That actually means a separate PR will be created with moving those right after this one is merged 😃